tests/components/application-connector/scripts/check-pod-logs.sh

```diff
@@ -8,8 +8,10 @@ if [ $# -ne 1 ]; then
   exit 1
 fi
 
+POD_NAME=$(kubectl get pods -n $NAMESPACE --selector=job-name=$JOB_NAME --output=jsonpath='{.items[*].metadata.name}')
+
 if ([[ ${EXPORT_RESULT} == true ]]); then
-	kubectl -n $NAMESPACE logs -f job/"$JOB_NAME" | tee /dev/stderr | $GOPATH/bin/go-junit-report -set-exit-code > junit-report.xml
+	kubectl -n $NAMESPACE logs $POD_NAME -f | tee /dev/stderr | $GOPATH/bin/go-junit-report -subtest-mode exclude-parents -set-exit-code > junit-report.xml
 else
-	kubectl -n $NAMESPACE logs -f job/"$JOB_NAME"
+	kubectl -n $NAMESPACE logs $POD_NAME application-gateway-test
```

Comment:
Is there any reason to do this instead of just getting logs from the job?

Reply:
I encountered an error when pulling logs straight from Job, through which kubectl ended up with an error. It is a sort of double check to make sure that the right logs are being pulled from the right place.